### PR TITLE
added social media icons

### PR DESCRIPTION
**Pull request by AbdullahiAbdulkabir**
Added social icons as said in #30 

image

### CODE REVIEW

File: src/partials/styles.html.eco [hunk 1]
```diff
@@ -5,5 +5,6 @@
 <link href="<%=@nonrooturl or ''%>css/style.css" rel="stylesheet">
 <link href="<%=@nonrooturl or ''%>css/contrast.css" rel="stylesheet">
 
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (sachin10101998) [inline src/partials/styles.html.eco]: Do we really need to add a new library to solve this issue? Bootstrap has an MIT license and the Fluid project has a Creative Commons Attribution License which may lead to conflicts. We should discuss the need of addition of a library and it should be added only if it's absolutely necessary.

Comment (AbdullahiAbdulkabir) [reply src/partials/styles.html.eco]: Fine, I will fix that.

**Review by Raunaksingh100 — CHANGES_REQUESTED** [review]
The issue #30   already closed , so no need of this

**Comment by AbdullahiAbdulkabir** [comment]
Closed in favour of the admin.

**Pull request closed without merging**